pkg/icsp/icsp.go

```diff
@@ -0,0 +1,252 @@
+package icsp
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/golang/glog"
+	"github.com/openshift-kni/eco-goinfra/pkg/clients"
+	"github.com/openshift-kni/eco-goinfra/pkg/msg"
+	v1alpha1 "github.com/openshift/api/operator/v1alpha1"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// ICSPBuilder provides struct for the ImageContentSourcePolicy object with connection to the cluster.
+type ICSPBuilder struct {
+	Definition *v1alpha1.ImageContentSourcePolicy
```

Comment:
Please add comments to all public vars of the struct

Reply:
Done